### PR TITLE
Added html for review

### PR DESCRIPTION
**Pull request by jaamanu**
-Added index.html for review
- committed new file for review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SURVEY PAGE</title>
+</head>
+<body>
+    <link rel="stylesheet" href="style.css">
```
Comment (jaamanu) [inline index.html:10]: Hi@BertrandConxy 
Good work done so far , you are almost there ! 
i will recommend few changes to finalise your project. 
Required Changes 
Please kindly put the style.css link inside the head tag.  
The link tag defines the relationship between the current document and an external resouces.
You can read more about html structure and how links are use using the following resouces https://www.sitepoint.com/a-basic-html5-template/  ,  (https://www.w3schools.com/tags/tag_link.asp),

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+<div class="header">
+    <h1 id="title"> Our Survey Form</h1>
+    <p>Thank you for taking the time to help us improve the platform</p>
+</div>
+
+    <div class="row">
+    <p id="description">Kenny and James survey-form :)</p>
+    <form action="web url" id="survey-form">
+    </form>
```
Comment (jaamanu) [inline index.html:20]: Please add the actual url link to the action attribute that will be directing the user input to a server.
The form closing tag should span across its content and all input element.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        <br><label><input type="checkbox" name="alice" value="alice">Alice</label>
+        <label><input type="checkbox" name="alice" value="kenny">Kenny</label>
+        <label><input type="checkbox" name="alice" value="date">Date</label>
+ 
+        <select title="option" name="Football Teams" id="dropdown">
+        <option value="Manchester United">Manchester United</option>
+        <option value="Chelsea">Chelsea</option>
+        </select>
+ 
+        <textarea title="text"></textarea>
```
Comment (jaamanu) [inline index.html:53]: Kindly use a placeholder in your textarea tag, which is a good practice for user-interaction to see the comment and what to do easily.

File: index.html (hunk 1 above)
Comment (BertrandConxy) [inline index.html:10]: - [ ] Good job so far 👏. Please **provide an apparent reason WHY you are suggesting this change**. You may consider providing links to some articles that explain that point very well 😉. 💡 Please edit the above comment which you have left before.

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @jaamanu, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


# Highlights 💪 
- Great use of GitHub review -- Inline comments feature
- Use polite phrases while giving reviews
- Provided a clear explanation of what changes to be made


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jaamanu** [comment]
Hi BertrandConxy,
Thank you for the information and correction, hope the new changes looks better than the first one although not perfect as i had wanted to put some icons and imojis but could not do so. Thanks

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html:10]: - [ ] @jaamanu Great work on making the changes requested by a previous reviewer 👏

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @jaamanu  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [x] Good Job for phrased the comments politely, as suggestions instead of orders.
- [x] Good Job for using git-hub inline comments to leave review comments.
- [x] Good job for providing Summarize for Requested changes 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jaamanu** [comment]
Hi@Abenezer
Thank you for the quick response  and review. 
James

**Pull request merged**